code/__DEFINES/preferences.dm

```diff
@@ -57,8 +57,9 @@
 #define PREFTOGGLE_2_FORCE_WHITE_RUNECHAT	1024
 #define PREFTOGGLE_2_SIMPLE_STAT_PANEL	2048
 #define PREFTOGGLE_2_SEE_ITEM_OUTLINES 	4096
+#define PREFTOGGLE_2_SEE_ITEM_TOOLTIPS  8192
```

Comment:
```suggestion
#define PREFTOGGLE_2_HIDE_ITEM_TOOLTIPS  8192
```
Enabling this turns it off, rename accordingly.

Reply:
yeah, makes more sense